### PR TITLE
Check timestamp with tz

### PR DESCRIPTION
**Pull request by rp280**

### CODE REVIEW

**Comment by SlowMo24** [comment]
I favour timestamps without time-zone. I never had a benefit only nuisance. For example you can note time zones differently (e.g. `z`== ` +00:00`, yet the ohsome api does not support all these notions. That is why we had to remove the `Z`in #127 

As you say: "everything is in UTC" so there is no benefit in marking that fact explicitly.

**Comment by rp280** [comment]
A Test for Issue #121 was also created in this Pull Request

File: ohsome/clients.py [hunk 1]
```diff
@@ -740,10 +740,30 @@ def count(self):
 class _OhsomeClientContributionsAggregated(_OhsomePostClient):
     @property
     def density(self):
-        return _OhsomePostClient(
+        return _OhsomeClientContributionsAggregatedDensity(
```
Comment (SlowMo24) [inline ohsome/clients.py:743]: please move these changes to the respective branch, then rebase on that branch. It is not a good idea to mix changes of different scope in one MR.

see #134

File: ohsome/response.py [hunk 2]
```diff
@@ -85,31 +85,38 @@ def _as_geodataframe(self, multi_index=True):
             )
 
         if "@validFrom" in features.columns:
+            features["@validFrom"] = features["@validFrom"].str.replace("Z", "")
```
Comment (SlowMo24) [inline ohsome/response.py:88]: move the str replacement within the other function, that would be cleaner

File: ohsome/response.py [hunk 3]
```diff
@@ -173,13 +180,16 @@ def _format_timestamp(self, result_df: DataFrame) -> None:
         :return:
         """
         if "timestamp" in result_df.columns:
+            result_df["timestamp"] = result_df["timestamp"].str.replace("Z", "")
             result_df["timestamp"] = pd.to_datetime(
                 result_df["timestamp"], format="ISO8601"
             )
         else:
+            result_df["fromTimestamp"] = result_df["fromTimestamp"].str.replace("Z", "")
```
Comment (SlowMo24) [inline ohsome/response.py:188]: same here and in all other occurences: move into function below

File: ohsome/test/test_response.py [hunk 4]
```diff
@@ -343,6 +344,25 @@ def test_contributions_latest(base_client):
     assert len(result) == 1
 
 
+def test_contributions_count_density_groupbyboundary(base_client):
```
Comment (SlowMo24) [inline ohsome/test/test_response.py:347]: move this test to the respective branch (see above)

**Review by SlowMo24 — CHANGES_REQUESTED** [review]
Nice progress. Please update the CHANGELOG and state explicitely that we are deviating from the ohsome api here

**Review by SlowMo24 — APPROVED** [review]

**Pull request merged**